packages/cli-template-contracts-hardhat/.eslintignore

```diff
@@ -0,0 +1,39 @@
+# dependencies
+node_modules
+package-lock.json
+yarn.lock
+.yarn
+
+# testing
+coverage
+coverage.json
+
+# hardhat
+artifacts
+cache
+typechain-types
+
+# types
+types
+
+# circuits
+circuits
+
+# production
+dist
+build
+docs
+
+# misc
+.DS_Store
+*.pem
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# packages
+cli-template-*
+
+snark-artifacts
```

Comment:
This folder shouldn't be part of this template anymore. So shouldn't be necessary to have it here